data/alibaba

```diff
@@ -34,6 +34,7 @@ alibabagroup.com
 alibabaplanet.com
 alibabaued.com
 alibabausercontent.com
+alicdn.com
```

Comment:
This domain exists in `aliyun`.

Reply:
You are right. This domain already exists in aliyun.